__test__/teacher.test.ts

```diff
@@ -21,6 +22,42 @@ const teacherToken = 'token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjE
 beforeEach(dbBuild);
 afterAll(() => dbConnection.end());
 
+const cookie = 'token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjMsInJvbGUiOiJ0ZWFjaGVyIiwiaWF0IjoxNjUwNzg1Mzk3fQ.ErDrf4OSpcRWmrQACekbQBNwCRbBo1dmdBHRDio9b4w;';
+
+describe('DELETE /api/v1/teacher/quiz/:quizId', () => {
+  it('should return 200 and all quizzes teacher data as json response', async () => {
+    const res = await supertest(app)
+      .delete('/api/v1/teacher/quiz/quiz-1')
+      .send(successDelete)
```

Comment:
You only need the `quizId` in order to delete a quiz so, there's no need to send this body, right?